test/core/consensus/grandpa/vote_graph/adjust_base.cpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "fixture.hpp"
+
+TEST_F(VoteGraphFixture, AdjustBase) {
+  BlockInfo base{6, "E"_H};
+  graph = std::make_shared<VoteGraphImpl>(base, chain);
+  auto &entries = graph->getEntries();
+
+  AssertGraphCorrect(
+      *graph,
+      R"({"entries":{"E":{"number":6,"ancestors":[],"descendents":[],"cumulative_vote":0}},"heads":["E"],"base":"E","base_number":6})");
+
+  expect_getAncestry("FB"_H, "FA"_H, "F"_H);
+  EXPECT_OUTCOME_TRUE_1(graph->insert(BlockInfo{10, "FC"_H}, "5"_W));
+  AssertGraphCorrect(
+      *graph,
+      R"({"entries":{"E":{"number":6,"ancestors":[],"descendents":["FC"],"cumulative_vote":5},"FC":{"number":10,"ancestors":["FB","FA","F","E"],"descendents":[],"cumulative_vote":5}},"heads":["FC"],"base":"E","base_number":6})");
```

Comment:
please format json's. Otherwise it is very hard to read. Also apply clang-format. If needed big strings could be under `clang-format -off`

Reply:
believe it or not, but it is already formatted. clang-format does not format raw string literals

Reply:
I mean format json with indents. There are plenty of tools online for doing so